politeiawww/cmsdatabase/cockroachdb/cockroachdb.go

```diff
@@ -69,6 +69,21 @@ func (c *cockroachdb) InvoicesByUserID(userid string) ([]database.Invoice, error
 		return nil, err
 	}
 
+	// XXX this could be done in a more efficient way
```

Comment:
If the `XXX this could be done in a more efficient way` comments were copied from the cache implementation, I'd probably take them out.  Looking back on it, I'm not even sure why I put it in there.